src/main/java/hudson/plugins/filesystem_scm/FSSCM.java

```diff
@@ -180,7 +183,17 @@ private boolean poll(AbstractProject<?, ?> project, Launcher launcher, FilePath
 		long start = System.currentTimeMillis();
 		
 		PrintStream log = launcher.getListener().getLogger();
-		log.println("FSSCM.pollChange: " + path);
+
+		String expandedPath = path;
+
+		AbstractBuild<?,?> lastCompletedBuild = project.getLastCompletedBuild();
+
+		if (lastCompletedBuild != null){

```

Comment:
I doubt that taking EnvVars from the previous build is an expected behavior for user if he uses Build Parameters in the declaration